examples/actix_example/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "sea-orm-actix-example"
+version = "0.1.0"
+authors = ["Sam Samai <sam@studio2pi.com.au>"]
+edition = "2018"
+publish = false
+
+[workspace]
+
+[dependencies]
+async-stream = { version = "^0.3" }
+async-trait = { version = "0.1" }
+futures = { version = "^0.3" }
+futures-util = { version = "^0.3" }
+actix-http = "2"
+actix-web = "3"
+actix-flash = "0.2"
+actix-files = "0.5"
+tera = "1.8.0"
+dotenv = "0.15"
+listenfd = "0.3.3"
+
+# remove `path = ""` in your own project
+sea-orm = { path = "../../", version = "^0.2", features = [
+  "macros",
+  "sqlx-all",
+  "runtime-async-std-native-tls",
```

Comment:
I am confused. It shouldnt compile? I thought we needed `runtime-actix-native-tls` ?